docs/topics/gradle.md

```diff
@@ -256,13 +256,13 @@ kotlin.stdlib.default.dependency=false
 
 ### Set dependencies on test libraries
 
-The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing different Kotlin projects. 
-
-Add the corresponding dependencies on test libraries:
-
-* For `commonTest`, add the `kotlin-test-common` and `kotlin-test-annotations-common` dependencies.
-* For JVM targets, use `kotlin-test-junit` or `kotlin-test-testng` for the corresponding asserter implementation and annotations mapping.
-* For Kotlin/JS targets, add `kotlin-test-js` as a test dependency. 
+The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing Kotlin projects on 
```

Comment:
See how this link looks: https://branch-kotlin-test-as-the-single-dependency.kotlin-web-site.labs.jb.gg/docs/gradle.html#set-dependencies-on-test-libraries

Should I edit the formatting somehow?

Reply:
Yes, let's move `API` out of the link.